constants/HallXCoordinates.js

```diff
@@ -1,234 +1,332 @@
 export function HallXCoordinates () {
```

Comment:
When you add *export* to a function, what does it add or do?

Reply:
export means i can import it anywhere, even in test cases which is why i need it